### PR TITLE
[JS] Add screenshot with Server Endpoint and API Key

### PR DESCRIPTION
**Pull request by TakLee96**
connects #158

### CODE REVIEW

**Comment by bensonby** [comment]
- better narrow your browser width before taking the screenshot
- The text is too blurred now (too small)
- The blur is still visible I guess?

**Comment by bensonby** [comment]
Also we will not use external image provider to host the image in this case

**Comment by TakLee96** [comment]
I think I will then just use the original image size (147KB is not so bad right?) and use CSS to resize it for pretty display. Also, how do we properly include a static image?

File: content/js/index.md [hunk 1]
```diff
@@ -12,6 +12,7 @@ application at the Skygear [Developer Portal](https://portal.skygear.io)
 website. After you registered, go to the **INFO** tab and copy down your
 `Server EndPoint` and `API Key`.
 
+![Example Image](/content/common/portal.png)
```
Comment (bensonby) [inline content/js/index.md]: The file name should be more specific. Imagine there can be more screenshots of the portal in other parts of the doc

File: content/js/index.md [hunk 2]
```diff
@@ -12,6 +12,7 @@ application at the Skygear [Developer Portal](https://portal.skygear.io)
 website. After you registered, go to the **INFO** tab and copy down your
 `Server EndPoint` and `API Key`.
 
+![Example Image](/assets/common/portal-endpoint-apikey.png)
```
Comment (bensonby) [inline content/js/index.md]: - A more accurate description other than "Example Image"?
- The image has to be clickable to enlarge it

**Pull request merged**